zcncore/wallet_mobile.go

```diff
@@ -22,6 +26,14 @@ func (w *wallet) Sign(hash string) (string, error) {
 	return sigScheme.Sign(hash)
 }
 
+func GetWalletBalanceMobile(id string) (string, error) {
+	balance, err := GetWalletBalance(id)
+	if err != nil {
+		return "", err
+	}
+	return strconv.FormatInt(int64(balance), 10), nil
```

Comment:
```suggestion
return int64(balance), nil
```

Reply:
last time you told that it is not working

Reply:
uint64 does't work on gomobile, but int64 works.

Reply:
fixed